### ISSUE TITLE
Package for 4.8.1 or later has Xcode hardwired in cmake files

### ISSUE THREAD

**Issue by agrossfield** [conda-forge/libnetcdf-feedstock#155]
### Solution to issue cannot be found in the documentation.

- [X] I checked the documentation.

### Issue

We're running into issues with building stuff depending on libnetcdf when using cmake on osx-arm64 and version 4.8.1 or later. We tracked the problem to netCDFTargets.cmake, which has a line that explicitly references Xcode: 
```
set_target_properties(netCDF::netcdf PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include;${_IMPORT_PREFIX}/include"
  INTERFACE_LINK_LIBRARIES "/Users/runner/miniforge3/conda-bld/libnetcdf_1664490027455/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libmfhdf.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1664490027455/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libdf.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1664490027455/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libhdf5_hl.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1664490027455/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libhdf5.dylib;/Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/usr/lib/libm.tbd;/Users/runner/miniforge3/conda-bld/libnetcdf_1664490027455/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libz.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1664490027455/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libzip.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1664490027455/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libzstd.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1664490027455/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libbz2.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1664490027455/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libcurl.dylib;/Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/usr/lib/libxml2.tbd"
)
```

This breaks our build on any Mac that doesn't have developer tools on it. It's not an issue on linux or before 4.8.1.

Ignore the conda list and conda info below -- I've fixed my environment by downgrading libnetcdf, and don't feel like rebreaking it.  I confirmed the problem by downloading osx-arm64/libnetcdf-4.9.0-mpi_mpich_h1ec8146_0.tar.bz2 and examining the cmake files directly.

### Installed packages

```shell
# packages in environment at /Users/agrossfield/miniconda3/envs/loos:
#
# Name                    Version                   Build  Channel
aoo-mozilla-en-dict-au    2022.08.01           h1a96a4e_0    conda-forge
aoo-mozilla-en-dict-ca    2022.08.01           h1a96a4e_0    conda-forge
aoo-mozilla-en-dict-gb    2022.08.01           h1a96a4e_0    conda-forge
aoo-mozilla-en-dict-us    2022.08.01           hd8ed1ab_0    conda-forge
aoo-mozilla-en-dict-za    2022.08.01           h1a96a4e_0    conda-forge
appnope                   0.1.2           py310h2ec42d9_2    conda-forge
asttokens                 2.0.5                    pypi_0    pypi
atk-1.0                   2.36.0               he69c4ee_4    conda-forge
backcall                  0.2.0                    pypi_0    pypi
backports                 1.0                        py_2    conda-forge
backports-functools-lru-cache 1.6.4                    pypi_0    pypi
backports.functools_lru_cache 1.6.4              pyhd8ed1ab_0    conda-forge
boost                     1.76.0          py310h509978a_1    conda-forge
boost-cpp                 1.76.0               hff03dee_1    conda-forge
bzip2                     1.0.8                h0d85af4_4    conda-forge
c-ares                    1.18.1               h0d85af4_0    conda-forge
c-compiler                1.3.0                h0d85af4_0    conda-forge
ca-certificates           2022.9.24            h033912b_0    conda-forge
cairo                     1.16.0            he43a7df_1008    conda-forge
cctools                   949.0.1             hd9211c8_21    conda-forge
cctools_osx-64            949.0.1             h6407bdd_21    conda-forge
clang                     11.1.0               h694c41f_1    conda-forge
clang-11                  11.1.0          default_he082bbe_1    conda-forge
clang-format              13.0.0          default_he082bbe_0    conda-forge
clang-format-13           13.0.0          default_he082bbe_0    conda-forge
clang_osx-64              11.1.0               hb91bd55_4    conda-forge
clangxx                   11.1.0          default_he082bbe_1    conda-forge
clangxx_osx-64            11.1.0               h7e1b574_4    conda-forge
cmake                     3.21.3               hdbd2f3e_0    conda-forge
compiler-rt               11.1.0               h654b07c_0    conda-forge
compiler-rt_osx-64        11.1.0               h8c5fa43_0    conda-forge
compilers                 1.3.0                h694c41f_0    conda-forge
curl                      7.80.0               h97da3c1_0    conda-forge
cxx-compiler              1.3.0                h940c156_0    conda-forge
debugpy                   1.6.3           py310h154be8b_0    conda-forge
decorator                 5.1.1                    pypi_0    pypi
doxygen                   1.9.3                h51faa4e_1    conda-forge
eigen                     3.4.0                h940c156_0    conda-forge
entrypoints               0.4                pyhd8ed1ab_0    conda-forge
executing                 0.8.3                    pypi_0    pypi
expat                     2.4.8                h96cf925_0    conda-forge
flake8                    4.0.1                    pypi_0    pypi
font-ttf-dejavu-sans-mono 2.37                 hab24e00_0    conda-forge
font-ttf-inconsolata      3.000                h77eed37_0    conda-forge
font-ttf-source-code-pro  2.038                h77eed37_0    conda-forge
font-ttf-ubuntu           0.83                 hab24e00_0    conda-forge
fontconfig                2.13.94              h10f422b_0    conda-forge
fonts-conda-ecosystem     1                             0    conda-forge
fonts-conda-forge         1                             0    conda-forge
fortran-compiler          1.3.0                h6818c3e_0    conda-forge
freetype                  2.10.4               h4cff582_1    conda-forge
fribidi                   1.0.10               hbcb3906_0    conda-forge
gdk-pixbuf                2.42.6               h2e6141f_0    conda-forge
gettext                   0.19.8.1          hd1a6beb_1008    conda-forge
gfortran                  9.3.0               h768ea0c_15    conda-forge
gfortran_impl_osx-64      9.3.0               h9cc0e5e_23    conda-forge
gfortran_osx-64           9.3.0               h18f7dce_15    conda-forge
giflib                    5.2.1                hbcb3906_2    conda-forge
git                       2.34.1          pl5321h2185044_0    conda-forge
gmp                       6.2.1                h2e338ed_0    conda-forge
graphite2                 1.3.13            h2e338ed_1001    conda-forge
graphviz                  2.50.0               h77de9ca_1    conda-forge
gtk2                      2.24.33              h675d97a_1    conda-forge
gts                       0.7.6                hccb3bdf_2    conda-forge
harfbuzz                  3.1.1                h159f659_0    conda-forge
hdf4                      4.2.15               hefd3b78_3    conda-forge
hdf5                      1.12.1          nompi_hd9e8a45_102    conda-forge
hunspell                  1.7.0             h2df5d5c_1001    conda-forge
hunspell-en               2022.08.01           h1a96a4e_0    conda-forge
icu                       68.2                 he49afe7_0    conda-forge
importlib-metadata        3.3.0                    pypi_0    pypi
ipykernel                 6.15.3             pyh736e0ef_0    conda-forge
ipython                   8.2.0           py310h2ec42d9_0    conda-forge
isl                       0.22.1               hb1e8313_2    conda-forge
jbig                      2.1               h0d85af4_2003    conda-forge
jedi                      0.18.1          py310h2ec42d9_0    conda-forge
joblib                    1.1.0                    pypi_0    pypi
jpeg                      9d                   hbcb3906_0    conda-forge
jupyter_client            7.3.5              pyhd8ed1ab_0    conda-forge
jupyter_core              4.11.1          py310h2ec42d9_0    conda-forge
krb5                      1.19.2               h289aae4_3    conda-forge
lapack                    3.9.0                    netlib    conda-forge
ld64                      530                 hd2e7500_21    conda-forge
ld64_osx-64               530                 he8994da_21    conda-forge
ldid                      2.1.2                h6a69015_3    conda-forge
lerc                      3.0                  he49afe7_0    conda-forge
libblas                   3.9.0           12_osx64_openblas    conda-forge
libcblas                  3.9.0           12_osx64_openblas    conda-forge
libclang-cpp11.1          11.1.0          default_he082bbe_1    conda-forge
libclang-cpp13            13.0.0          default_he082bbe_0    conda-forge
libcurl                   7.80.0               h97da3c1_0    conda-forge
libcxx                    14.0.5               hce7ea42_1    conda-forge
libdeflate                1.8                  h0d85af4_0    conda-forge
libedit                   3.1.20191231         h0678c8f_2    conda-forge
libev                     4.33                 haf1e3a3_1    conda-forge
libffi                    3.4.2                h0d85af4_5    conda-forge
libgd                     2.3.3                h311d3fc_0    conda-forge
libgfortran               5.0.0           9_3_0_h6c81a4c_23    conda-forge
libgfortran-devel_osx-64  9.3.0               h6c81a4c_23    conda-forge
libgfortran5              9.3.0               h6c81a4c_23    conda-forge
libglib                   2.70.2               hf1fb8c0_4    conda-forge
libiconv                  1.16                 haf1e3a3_0    conda-forge
liblapack                 3.9.0           12_osx64_openblas    conda-forge
libllvm11                 11.1.0               hd011deb_2    conda-forge
libllvm13                 13.0.0               hd011deb_0    conda-forge
libnetcdf                 4.8.1           nompi_h6609ca0_101    conda-forge
libnghttp2                1.43.0               hfd382f3_1    conda-forge
libopenblas               0.3.18          openmp_h3351f45_0    conda-forge
libpng                    1.6.37               h7cec526_2    conda-forge
librsvg                   2.52.5               h02b643e_1    conda-forge
libsodium                 1.0.18               hbcb3906_1    conda-forge
libssh2                   1.10.0               hd3787cc_2    conda-forge
libtiff                   4.3.0                hd146c10_2    conda-forge
libtool                   2.4.6             he49afe7_1008    conda-forge
libuv                     1.44.2               hac89ed1_0    conda-forge
libwebp                   1.2.2                h28dabe5_0    conda-forge
libwebp-base              1.2.2                h0d85af4_1    conda-forge
libxml2                   2.9.12               h93ec3fd_0    conda-forge
libzip                    1.8.0                h7e5727d_1    conda-forge
libzlib                   1.2.11            h9173be1_1013    conda-forge
llvm-openmp               12.0.1               hda6cdc1_1    conda-forge
llvm-tools                11.1.0               hd011deb_2    conda-forge
loos                      4.0.0                    pypi_0    pypi
lz4-c                     1.9.3                he49afe7_1    conda-forge
matplotlib-inline         0.1.3                    pypi_0    pypi
mccabe                    0.6.1                    pypi_0    pypi
mpc                       1.2.1                hbb51d92_0    conda-forge
mpfr                      4.1.0                h0f52abe_1    conda-forge
ncurses                   6.2                  h2e338ed_4    conda-forge
nest-asyncio              1.5.5              pyhd8ed1ab_0    conda-forge
ninja                     1.11.0               h1b54a9f_0    conda-forge
numpy                     1.21.4          py310hd5a656b_0    conda-forge
openbabel                 3.1.1           py310he467037_3    conda-forge
openblas                  0.3.18          openmp_h12da7db_0    conda-forge
openssl                   3.0.5                hfd90126_2    conda-forge
packaging                 21.3               pyhd8ed1ab_0    conda-forge
pango                     1.48.10              h056538c_2    conda-forge
parso                     0.8.3                    pypi_0    pypi
pcre                      8.45                 he49afe7_0    conda-forge
pcre2                     10.37                ha16e1b2_0    conda-forge
perl                      5.32.1          2_h0d85af4_perl5    conda-forge
pexpect                   4.8.0                    pypi_0    pypi
pickleshare               0.7.5                    pypi_0    pypi
pip                       21.3.1                   pypi_0    pypi
pixman                    0.40.0               hbcb3906_0    conda-forge
prompt-toolkit            3.0.27                   pypi_0    pypi
psutil                    5.9.2           py310h90acd4f_0    conda-forge
ptyprocess                0.7.0                    pypi_0    pypi
pure-eval                 0.2.2                    pypi_0    pypi
pure_eval                 0.2.2              pyhd8ed1ab_0    conda-forge
pycodestyle               2.8.0                    pypi_0    pypi
pyflakes                  2.4.0                    pypi_0    pypi
pygments                  2.11.2                   pypi_0    pypi
pyparsing                 3.0.9              pyhd8ed1ab_0    conda-forge
python                    3.10.0          h38b4d05_3_cpython    conda-forge
python-dateutil           2.8.2              pyhd8ed1ab_0    conda-forge
python_abi                3.10                    2_cp310    conda-forge
pyzmq                     24.0.0          py310hf615a82_0    conda-forge
readline                  8.1                  h05e3726_0    conda-forge
rhash                     1.4.3                hac89ed1_0    conda-forge
scikit-learn              1.0.1           py310h598de7d_2    conda-forge
scipy                     1.7.3           py310h47774c9_0    conda-forge
scons                     4.3.0           py310h2ec42d9_0    conda-forge
setuptools                59.4.0          py310h2ec42d9_0    conda-forge
six                       1.16.0                   pypi_0    pypi
sqlite                    3.37.0               h23a322b_0    conda-forge
stack-data                0.2.0                    pypi_0    pypi
stack_data                0.2.0              pyhd8ed1ab_0    conda-forge
swig                      4.0.2                hce5123c_2    conda-forge
tapi                      1100.0.11            h9ce4665_0    conda-forge
threadpoolctl             3.0.0                    pypi_0    pypi
tk                        8.6.11               h5dbffcc_1    conda-forge
tornado                   6.2             py310h6c45266_0    conda-forge
traitlets                 5.1.1                    pypi_0    pypi
typing-extensions         4.0.1                    pypi_0    pypi
typing_extensions         4.0.1              pyha770c72_0    conda-forge
tzdata                    2021e                he74cb21_0    conda-forge
uncrustify                0.74.0               h96cf925_0    conda-forge
wcwidth                   0.2.5                    pypi_0    pypi
wheel                     0.37.0                   pypi_0    pypi
xz                        5.2.5                haf1e3a3_1    conda-forge
zeromq                    4.3.4                he49afe7_1    conda-forge
zipp                      3.6.0                    pypi_0    pypi
zlib                      1.2.11            h9173be1_1013    conda-forge
zstd                      1.5.0                h582d3a0_0    conda-forge
```

### Environment info

```shell
active environment : loos
    active env location : /Users/agrossfield/miniconda3/envs/loos
            shell level : 1
       user config file : /Users/agrossfield/.condarc
 populated config files : /Users/agrossfield/.condarc
          conda version : 4.14.0
    conda-build version : not installed
         python version : 3.9.5.final.0
       virtual packages : __osx=10.16=0
                          __unix=0=0
                          __archspec=1=x86_64
       base environment : /Users/agrossfield/miniconda3  (writable)
      conda av data dir : /Users/agrossfield/miniconda3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/osx-64
                          https://conda.anaconda.org/conda-forge/noarch
                          https://repo.anaconda.com/pkgs/main/osx-64
                          https://repo.anaconda.com/pkgs/main/noarch
                          https://repo.anaconda.com/pkgs/r/osx-64
                          https://repo.anaconda.com/pkgs/r/noarch
          package cache : /Users/agrossfield/miniconda3/pkgs
                          /Users/agrossfield/.conda/pkgs
       envs directories : /Users/agrossfield/miniconda3/envs
                          /Users/agrossfield/.conda/envs
               platform : osx-64
             user-agent : conda/4.14.0 requests/2.28.1 CPython/3.9.5 Darwin/21.6.0 OSX/10.16
                UID:GID : 1326343926:617926151
             netrc file : None
           offline mode : False
```

**Comment by agrossfield** [comment]
I should be clear -- it's not just that it mentions Xcode, it's that it's hardwired for a specific version of Xcode and the SDK. This will fail for almost everyone.

**Comment by xylar** [comment]
Hi @agrossfield, since no one else has chimed in, I'll respond.  First, I'm not a OSX expert and my CMake experience is also pretty limited.  So take what I say with a grain of salt.

That being said, I think the libraries on conda-forge are largely intended for compatibility within its own ecosystem, rather than for direct use by others.  If you need libnetcdf for your own projects, conda-forge is likely not the place to get it from.  If you are trying to build conda-forge packages locally, you are likely best off getting the same docker images as conda-forge uses for CI and building with them.  I don't know whether that will help for OSX but it makes a difference on both Windows and Linux.

@conda-forge/libnetcdf, anyone else have thoughts?  Is there something easy we can do to generalize our CMake files?

**Comment by agrossfield** [comment]
I can understand that, but this still seems like an error. One of the libraries is pulled from Xcode libm -- I find it hard to believe that the one provided by the conda compilers, etc, doesn't work. The other is libxml2, and again I'm 99% sure you can get that from conda as well. I'm pretty sure you don't want the cmake files to reference anything outside the conda ecosystem unless it's 100% necessary.

The reason this matters for me is that I distribute a package that people some people will want to compile for themselves -- basically, it's simultaneously suite of tools and a library for making new ones. We have a brand new conda-forge recipe for people who are just using the existing tools, but people who are either modifying the library or writing new c++ tools will want an actual source tree. To make this easier to support, we've given people a script that sets up an appropriate conda environment and builds the tool for them. As a result, on Mac your cmake scripts will only work if they have the exact same version of Xcode that you do.

This change seems to be new to version 4.9 on intel Mac -- earlier versions don't make any reference to Xcode -- so we were able to work around it by pinning to earlier versions, but on arm Mac there is no earlier version that works, so we're up a creek.

If this change was deliberate, by all means keep it, and we'll figure out a workaround. However, if it were just an accident, it would be great to fix it. Can someone use git blame to figure out who put the change in?

**Comment by xylar** [comment]
@agrossfield, I agree with you that it doesn't sound like we should be pulling in libraries like libm and libxml2 from Xcode.  As you say, they should be coming from conda-forge.

I don't think it's quite a straightforward as using git blame to figure out who changed things here.  I had a look at all the changes made in the various builds of 4.8.1:
* https://github.com/conda-forge/libnetcdf-feedstock/pull/129/files
* https://github.com/conda-forge/libnetcdf-feedstock/pull/131/files
* https://github.com/conda-forge/libnetcdf-feedstock/pull/137/files
Nothing we changed here jumped out at me as a likely culprit.  There could be changes in libnetcdf itself (from upstream https://github.com/Unidata/netcdf-c), from CMake or from dependencies on conda-forge that caused this.

@dopplershift, do you have any insights?  Anyone we could bring in to have a look?

**Comment by agrossfield** [comment]
I think I found the offending line, in build.sh (git blame says the change was introduced by 
b899096d (Isuru Fernando                                     2021-04-09 14:53:35 -0500 102)     

```
sed -i.bak "s#/Applications/Xcode_.*app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.*sdk/usr/lib/lib\([a-z]*\).dylib#\1#g" ${fname}
```

The pattern should be `Xcode_*.*app`, not `Xcode_.*app`, since the pattern it's supposed to be removing is `Xcode_13.2.1.app`

**Comment by xylar** [comment]
@isuruf, could you take a look at this? I know you're very busy with python 3.11 work but I have a feeling you're going to be able to understand what was intended with this line and how to fix it much more quickly than anyone else.

@agrossfield, thank you for this detective work!  It looks like most builds of 4.8.0 ware also affected.  I think if you put in a PR to try to fix this, we could get feedback from @isuruf before it goes in.

**Comment by xylar** [comment]
Hmm, @agrossfield, looking into this further, I think @isuruf only changed indentation on that particular line and it was introduced in https://github.com/conda-forge/libnetcdf-feedstock/pull/114 by @jschueller.

@jschueller, do you have any feedback on how to fix this line?  See:
https://github.com/conda-forge/libnetcdf-feedstock/pull/114/files#diff-d7075654874cb08007a21aaab3ecd4b3453a9087e7505d034d548b8938b599bc

**Comment by dopplershift** [comment]
(optional) libxml2 support was added in 4.8. Looks like we haven't updated the recipe to do anything about it, but on macOS it's picking up a system copy.

**Comment by xylar** [comment]
Okay, so it sounds like `libxml2` would be good to add explicitly so OSX doesn't find its own copy. Any intuition on the `libm` thing? I that just also something we should be bringing in explicitly?

**Comment by agrossfield** [comment]
There is an openlibm package, but in my experience that shouldn't be necessary -- although there's no $CONDA_PREFIX/lib/libm.so (or dylib) supplied by conda, from what I can tell the compilers do the right thing without need for an external package.

**Comment by dopplershift** [comment]
`libm` should be coming from the compiler/C runtime's standard library. That shouldn't be a system copy.

**Comment by xylar** [comment]
@agrossfield, this auto-closed in https://github.com/conda-forge/libnetcdf-feedstock/pull/156 but please check to see if it is actually fixed (once the new build is available in an hour or so).

**Comment by xylar** [comment]
We can apply the same fix to 4.8.1 if need be.

**Comment by xylar** [comment]
In fact, I'm going to go ahead and fix 4.8.1 the same way.

**Comment by xylar** [comment]
https://github.com/conda-forge/libnetcdf-feedstock/pull/157

**Comment by agrossfield** [comment]
The fix didn’t work. You now have the equivalent problem on Linux (I’m told by someone who just couldn’t build our software). 

Someone needs to examine the netCDFTargets.cmake as it gets installed.

**Comment by xylar** [comment]
Okay, that's disappointing but I don't have any more time to put into this and in any case it's well beyond my expertise. Again a PR would be welcome.

**Comment by ihnorton** [comment]
A variant of this is hopefully fixed in #169:

- from `lib/cmake/netCDF/netCDFTargets.cmake` in [osx-arm64/libnetcdf-4.9.2-mpi_mpich_h7298f9c_0.conda](https://anaconda.org/conda-forge/libnetcdf/4.9.2/download/osx-arm64/libnetcdf-4.9.2-mpi_mpich_h7298f9c_0.conda):
```
set_target_properties(netCDF::netcdf PROPERTIES
  INTERFACE_INCLUDE_DIRECTORIES "${_IMPORT_PREFIX}/include;${_IMPORT_PREFIX}/include"
  INTERFACE_LINK_LIBRARIES "/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libmfhdf.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libdf.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libhdf5_hl.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libhdf5.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libcrypto.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libcurl.dylib;/Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/usr/lib/libpthread.tbd;/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libsz.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libz.dylib;/Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/usr/lib/libdl.tbd;/Applications/Xcode_13.2.1.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX11.0.sdk/usr/lib/libm.tbd;/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libzstd.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libbz2.dylib;/Users/runner/miniforge3/conda-bld/libnetcdf_1678902346966/_h_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold/lib/libxml2.dylib"
)

```
